docs/product/dev-toolbar/setup.mdx

```diff
@@ -0,0 +1,144 @@
+| `domId (optional)` | `string \| undefined` | The \`id\` given to the \<div\> that is created to contain the toolbar html. | `"sentry-toolbar"` |
+| `debug (optional)` | `string \| undefined`  | A comma separated string of debug targets to enable.  Example: `"logging,state"`  If the list contains ”all” or ”true” then all targets will be enabled. Valid targets: `"logging" "login-success" "settings" "state"` | `undefined` |
+| `mountPoint (optional)` | `HTMLElement \| () => HTMLElement \| undefined` | Where to mount the toolbar in the DOM. | `document.body` |
+
+
+## Deploying To Production & Dev Environments
+Since the Dev Toolbar is deployed onto specific pages, it's strongly recommended that you consider which environments your app is deployed to should have the toolbar available.
+
+In dev and staging environments, you might want to *unconditionally* include the toolbar so that all developers and testers can use it and quickly go from the page they're looking at back to Sentry for further debugging.
+
+In production however, it's strongly recommended to *conditionally* include the Dev Toolbar `<script>` tag so that only members of your Sentry organization can see it. The specific implementation for conditionally including the Dev Toolbar is something you'll need to write based on how your app works and how your dev team is set up.
+
+For example, if your web application requires authentication to access, you could add a conditional where the Dev Toolbar is shown always when deployed to development **and** staging and is shown only if an employee is logged in when deployed to production.
+
+
+The code might look like this:
+```javascript
+// example conditions to render the toolbar in different environments.
+
+const env = process.env.SENTRY_ENVIRONMENT || 'development';
+const isEmployee = user.email.endsWith('@joshys-pizza.com')
+
+const isDev = env === 'development' || env === staging;
+const isEmployeeInProd = env === 'production' && isEmployee;
+if (isDev || isEmployeeInProd) {
+  SentryToolbar.init({ ... });
+}
+```
+
+If the toolbar `<script>` is accidentally included on your site, and `SentryToolbar.init()` is called, then a "Login to Sentry" button will be visible to the public. This is not ideal, but your data in Sentry will still be safe as users outside of your Sentry organization will not be able to authenticate themselves.
+
+## Conditionally Inserting Script Tag
+
+It's possible to dynamically insert the script tag inside a single-page app, prior to calling `SentryToolbar.init()`, so that it’s only visible to authorized users. See [`docs/conditional-script.md`](http://conditional-script.md) for example code. This will help reduce network traffic for your users because they do not have the credentials needed to login.
```

Comment:
I'm assuming we just haven't made this page yet?

Reply:
This was copy+pasted from the readme, the link could point to the repo where that file does exist. @c298lee can you update this link and check for any others that are broken or wrong?